src/providers/data_provider_be.c

```diff
@@ -260,11 +261,19 @@ static void be_check_online_done(struct tevent_req *req)
         goto done;
     }
 
-    switch (reply.dp_error) {
+    switch (reply->dp_error) {
     case DP_ERR_OK:
+        if (last_dp_state != DP_ERR_OK) {
+            last_dp_state = DP_ERR_OK;
+            sss_log(SSS_LOG_INFO, "Backend is online\n");
```

Comment:
Does SSS_LOG_INFO turn up in the syslog by default on RHEL-7?

Reply:
Hmm. That's a good question, I'll check it soon and change it in case it's needed.

Reply:
What is a answer to Jakub's question?

Reply:
@lslebodn, you have missed https://github.com/SSSD/sssd/pull/404#issuecomment-336918930

There I've said (and I'm quoting here): "tested with RHEL-7 and confirmed that SSS_LOG_INFO is enough"